docker/docker-compose.features.yml

```diff
@@ -4,4 +4,4 @@ services:
   dicomserver:
     environment:
       DicomServer__Features__EnableDataPartitions: "true"
-      DicomServer__Features__EnableDropInvalidDicomJsonMetadata: "true"
+      DicomServer__Features__EnableLatestApiVersion: "true"
```

Comment:
I realize this isn't your naming convention, but the `EnableLatestApiVersion` doesn't sound very exact. Especially because the `DicomApiVersion.cs` has a const called `Latest` which is set to `v1`.

Reply:
Sounds good to me. I think also it allows us to have a cleaner path to cleanup once `v2` is enabled because there's no question on the use of the flag.